### DISCUSSION TITLE
Running a Python Command (I'm a complete newbie and need help)

### DISCUSSION THREAD

**Discussion by haroldlibbert** [community/community#125715] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need to run this command to export these emojis. I'm lost. I'm not a coder by any chance. I am on a Mac. Just need some help. I own emojistickers.com. I will send out a free sticker pack to anyone who helps me. Thanks!

https://github.com/akfreas/emoji-extractor-plus

**Comment by Abhishek22K** [comment]
Install Python (if not already installed)
Install the emoji Library (pip3 install emoji)
Create a Python script to export emojis.
Run the script and check the emojis.txt file.
